suite-common/graph/src/graphDataFetching.ts

```diff
@@ -65,8 +96,14 @@ export const getAccountBalanceHistory = async ({
         return accountBalanceHistoryCache[cacheKey];
     }
 
-    const [accountMovementHistory, accountInfo] = await Promise.all([
-        TrezorConnect.blockchainGetAccountBalanceHistory({
+    const getBalanceHistory = async () => {
```

Comment:
Two simillar functions with the same name `getBalanceHistory` are defined in both `graphDataFetching.ts` and `graphBalanceEvents.ts`. Could not it be defined outside of these files and reused?

Reply:
They should be replaced soon by fetching everything from TransactionCache so I don't want to invest more energy into that.